ui/app/components/app/signature-request/signature-request-message/index.scss

```diff
@@ -45,22 +39,23 @@
 
   &--node,
   &--node-leaf {
-    padding-left: 0.8rem;
+    padding-left: 0.4rem;
 
     &-label {
       color: #5b5d67;
+      font-weight: 600;
     }
 
     &-value {
       color: black;
-      margin-left: 0.5rem;
-      text-overflow: ellipsis;
-      white-space: nowrap;
       overflow: hidden;
+      word-wrap: break-word;
     }
   }
 
   &--node-leaf {
-    display: flex;
+    span {
+      display: block;
```

Comment:
Actually, just read this again. Why is this span set to `display: block;`? I mean what effect were you trying to achieve.

A display property might not be needed here? And span's with `display: block;` are basically divs, so if this is necessary it would be better to just change those spans to divs and remove this extra css rule.

Although, I suspect that this was trying to achieve a specific effect, so perhaps something else is needed.